INSTALL.md

```diff
@@ -189,6 +189,9 @@ brew install openssl@1.1
 export LDFLAGS=-L/usr/local/Cellar/openssl\@1.1/1.1.1d/lib/
 ```
 
+Alternatively, you can try https://stackoverflow.com/a/56228387/796832
```

Comment:
Was it that you needed to also configure `CPPFLAGS` or was the path wrong?

Reply:
@clokep I'm not sure which one. After copy-pasting both of of those lines linked in the answer, it worked.

I did have the `openssl@1.1` brew package installed.

Reply:
Looks like homebrew puts symlinks into `/usr/local/opt` that (by default) point to /usr/local/Cellar. What version of openssl do you have installed? I now have `1.1.1i` so I'm guessing the issue was that the above command directly specified the `1.1.1d`? I think we should check the above directions to what the SO answer suggests (no need to link to it IMO):

```
brew install openssl@1.1
export LDFLAGS="-L/usr/local/opt/openssl/lib"
export CPPFLAGS="-I/usr/local/opt/openssl/include"
```

Reply:
I should note that this is just a symlink 😄 :

```
ls -l /usr/local/opt/openssl*   
lrwxr-xr-x  1 clokep  admin  28 Dec 14 09:12 /usr/local/opt/openssl -> ../Cellar/openssl@1.1/1.1.1i
lrwxr-xr-x  1 clokep  admin  28 Dec 14 09:12 /usr/local/opt/openssl@1.1 -> ../Cellar/openssl@1.1/1.1.1i
```

Reply:
:+1:  that looks great to me! Thanks for digging to find the symlinks are the same anyway 😀

Looks like I have the same version:
```
$ brew info openssl@1.1
openssl@1.1: stable 1.1.1i (bottled) [keg-only]
```